sprout/views/projects_id_metadata_create.py

```diff
@@ -66,30 +64,30 @@ def handle_post_request_with_file(
         validate_csv_and_save_columns(table_id, file_meta)
     except csv.Error as csv_error:
         file_meta.delete()
-        return render_metadata_create_page(request, table_id, csv_error.args[0])
+        return render_projects_id_metadata_create(request, table_id, csv_error.args[0])
 
-    return redirect("/column-review/" + str(table_id))
+    return redirect("/metadata/" + str(table_id) + "/update")
 
 
-def render_metadata_create_page(
+def render_projects_id_metadata_create(
     request: HttpRequest, table_id: int, upload_error: str
 ) -> HttpResponse:
-    """Render metadata/create page with an error if there is any.
+    """Render page with an error if there is any.
 
     Args:
         request: The http request
         table_id: The id of the table
         upload_error: The error message if there is any
 
     Returns:
-        HttpResponse: A html page based on the metadata-create.html template
+        HttpResponse: A html page based on the template
     """
     table_metadata = TableMetadata.objects.get(pk=table_id)
-    metadata_create_data = {
+    data_metadata = {
```

Comment:
Is this just `metadata`? If yes, change on L90 as well
```suggestion
    metadata = {
```

Reply:
This probably should be called `context` since that is what it is used in `render()`.